enterprise/server/usage/usage.go

```diff
@@ -83,19 +83,31 @@ func NewFlushLock(env environment.Env) interfaces.DistributedLock {
 	return redisutil.NewWeakLock(env.GetDefaultRedisClient(), redisUsageLockKey, redisUsageLockExpiry)
 }
 
+type TrackerOpts struct {
+	// Region identifies the datacenter in which the Redis deployment is running.
+	// Usage is tracked separately per Redis deployment.
+	Region string
+}
+
 type tracker struct {
-	env   environment.Env
-	rdb   *redis.Client
-	clock timeutil.Clock
+	env    environment.Env
+	rdb    *redis.Client
+	clock  timeutil.Clock
+	region string
 
 	flushLock interfaces.DistributedLock
 	stopFlush chan struct{}
 }
 
-func NewTracker(env environment.Env, clock timeutil.Clock, flushLock interfaces.DistributedLock) *tracker {
+func NewTracker(env environment.Env, clock timeutil.Clock, flushLock interfaces.DistributedLock, opts *TrackerOpts) *tracker {
+	region := opts.Region
+	if region == "" {
+		log.Warningf("Region is not configured; usage may not be tracked properly in multi-region deployments.")
```

Comment:
do you want to write data in this scenario?

Reply:
Probably not -- turned this into a fatal error in `main.go` instead.